tests/data/valid/transaction/null_values.json

```diff
@@ -22,6 +22,11 @@
         "architecture": null,
         "platform": null
     },
+    "process": {
+        "pid": null,
```

Comment:
again - this doesn't make sense to me - either `process.pid` is required, or it is allowed to be null.